kdb/key_test.go

```diff
@@ -61,11 +61,14 @@ func TestBoolean(t *testing.T) {
 	Assertf(t, val == testValue, "Key.Boolean() %t did not match %t", val, testValue)
 }
 
+// TODO REVIEW: Not all types tested?
```

Comment:
Bytes + String + Boolean are all tested, which one is missing?

Reply:
Later I saw that int/float are not yet implemented: Yes, so all implemented are tested.